packages/opencensus-web-core/src/trace/model/root-span.ts

```diff
@@ -46,7 +48,7 @@ export class RootSpan extends Span implements coreTypes.RootSpan {
     }
   }
 
-  startChildSpan(name?: string, kind?: string): Span {
+  startChildSpan(name?: string, kind?: SpanKind): Span {
```

Comment:
May be in another PR, we should change ```startChildSpan``` signature to accept object instead of two params (similar to ```startRootSpan```).

Reply:
Sure, I can do that. Added it here in this PR.